src/cpu/core_dynrec/cache.h

```diff
@@ -269,12 +254,7 @@ class CodePageHandlerDynRec : public PageHandler {
 				invalidation_map=(Bit8u*)malloc(4096);
 				memset(invalidation_map,0,4096);
 			}
-#if defined(WORDS_BIGENDIAN) || !defined(C_UNALIGNED_MEMORY)
-			host_writed(&invalidation_map[addr],
-				host_readd(&invalidation_map[addr])+0x1010101);
-#else
-			(*(Bit32u*)&invalidation_map[addr])+=0x1010101;
```

Comment:
PPC: Fix needs to be aligned and native endian (not swapped to little-endian like the host_* functions do).